airflow-core/tests/unit/models/test_mappedoperator.py

```diff
@@ -334,7 +334,12 @@ def execute(self, context):
 
 
 def _create_mapped_with_name_template_taskflow(*, task_id, map_names, template):
-    from airflow.providers.standard.operators.python import get_current_context
+    from tests_common.test_utils.version_compat import AIRFLOW_V_3_0_PLUS
+
+    if AIRFLOW_V_3_0_PLUS:
+        from airflow.sdk import get_current_context
+    else:
+        from airflow.providers.standard.utils import get_current_context
```

Comment:
```suggestion
    from airflow.sdk import get_current_context
```

Reply:
Hmm... some of those tests failed in backcompat test - but maybe it was the sagemaker - let me see